lib/experimental/synchronization.php

```diff
@@ -32,7 +32,16 @@ function gutenberg_add_collaborative_editing_post_type_support() {
 		return;
 	}
 
-	foreach ( array( 'page', 'post' ) as $post_type ) {
+	/*
+	 * Filter the post types that support collaborative editing.
+	 * By default, only 'post' and 'page' support it.
+	 *
+	 * @param array $post_types Array of post type names.
+	 * @return array Filtered array of post type names.
+	 */
+	$post_types = apply_filters( 'collaborative_editing_post_types', array( 'page', 'post' ) );
```

Comment:
Currently, we have instructions for this in the RTC plugin's README to do the following to bake in this support for other post types: 

`add_post_type_support( $post_type, 'collaborative-editing' )`

That ties into how we have been relying on what the post supports to drive the collaboration related pieces.

Personally, I like that more and would instead push to update whatever docs we can to add the above instruction in. It'd be better to have it Gutenberg rather than in our RTC plugin.